### ISSUE TITLE
Training schedule

### ISSUE THREAD

**Issue by huangdi95** [romulus0914/MixNet-Pytorch#6]
Hi, have you trained mixnet by yourself? Could you please release the training code.
I trained it with lr=0.016, warmup epoch=5, exponential decay and rmsprop optimizer, but failed. The acc-1 stays at 20%.

**Comment by ansleliu** [comment]
@huangdi95 you can download all mixnet pre-trained weights from this repo.: [https://github.com/ansleliu/MixNet-Pytorch](https://github.com/ansleliu/MixNet-Pytorch)
